src/main/java/org/mamute/dao/QuestionDAO.java

```diff
@@ -278,5 +277,126 @@ public Question questionWith(Attachment attachment) {
 				.setParameter("a", attachment)
 				.uniqueResult();
 	}
-}
 
+	public void delete(Question question) {
```

Comment:
I don't like the idea of really removing a row from bd, it sounds too risky, maybe its just a different deletion where moderators/owners can't see the questions either(and owners can delete theirs)